doc/gallery/introduction/pytensor_intro.ipynb

```diff
@@ -0,0 +1,3311 @@
+{
```

Comment:
<p>We shouldn't need to set autoreload (it misbehaves with pytensor anyway)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pytensor/pull/1585/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>